ui/app/templates/components/transit-key-action/hmac.hbs

```diff
@@ -37,11 +46,24 @@
         <label for="input" class="is-label">
           Input
         </label>
-        <div class="control is-relative">
-          {{textarea id="input" name="input" value=input class="textarea" data-test-transit-input="input"}}
-          {{b64-toggle value=input isInput=false data-test-transit-b64-toggle="input"}}
+        <div id="input-control" class="control is-relative">
+          {{ivy-codemirror
+            value=input
+            valueUpdated=(action (mut input))
+            options=(hash
+              lineNumbers=true
+              tabSize=2
+              mode='ruby'
+              theme='hashi'
+            )
+            data-test-transit-input="input"
+          }}
         </div>
       </div>
+      <div class="field">
+        {{input type="checkbox" id="encodedBase64" checked=encodedBase64 data-test-transit-input="encodedBase64"}}
```

Comment:
could you add some linebreaks and indentation here as well? :D if you want you can install the [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [EditorConfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) extensions which i _think_ will autofix everything to meet our linting standards inside our [.editorconfig](https://github.com/hashicorp/vault/blob/master/ui/.editorconfig).

Reply:
🤦‍♀ My workspace was based in the wrong folder, so it wasn't reading the config file. Fixed now, thank you for reminding me!